### DISCUSSION TITLE
BWCB Webinar Instructions

### DISCUSSION THREAD

**Discussion by jasminebro** [KarchinLab/open-cravat#271] [General]
Thank you for registering for the webinar! Please download OpenCravat to your laptop from: **https://open-cravat.readthedocs.io/en/latest/1.-Installation-Instructions.html** to follow along during the webinar.

Links to data files will also be posted on this discussion thread! Feel free to comment on this thread with questions or if you have issues installing.

**Comment by jasminebro** [comment]
**Pancreatic Cancer Zip Files**
Yachida, Shinichi et al. “Distant metastasis occurs late during the genetic evolution of pancreatic cancer.” Nature vol. 467,7319 (2010): 1114-7. doi:10.1038/nature09515

Please download and unzip files before the webinar. Thank you! 

[A32.zip](https://github.com/user-attachments/files/15930232/A32.zip)
[A38.zip](https://github.com/user-attachments/files/15930233/A38.zip)

**Reply by RachelKarchin** [reply]
Thanks so much to Drs. Christine Iacobuzio-Donahue (MSKCC) and James Eshleman (JHMI) for sharing their data with us.

**Comment by jasminebro** [comment]
**Annotators to Install from the OpenCravat Store after you have OpenCravat Installed:**
1. launch OpenCRAVAT by opening a terminal on your laptop and typing  
   oc gui
2. Go to the Store tab


3. You can install an annotator by first looking it up by name in the search box

Then Click on the Install button in the tile for the Annotator


4. For the workshop, you will install these 12 annotators:
Cancer Hotspots
Cancer Gene Census
Cancer Gene Landscapes
CIViC
ClinGen Gene
ClinVar ACMG
COSMIC
dbSNP 
GeneHancer
gnomAD3
Gene Ontology
LitVar

**Comment by jasminebro** [comment]
![OpenCRAVAT BWCB Webinar (1)](https://github.com/KarchinLab/open-cravat/assets/8697445/e3298b9a-c174-4682-ac89-46af5b982ff4)
